package/kedro_viz/server.py

```diff
@@ -56,18 +59,17 @@
     from kedro.framework.cli import get_project_context
     from kedro.framework.cli.utils import KedroCliError
 else:
-    # pylint: disable=no-name-in-module,import-error
     from kedro.cli import get_project_context  # pragma: no cover
-
-    # pylint: disable=no-name-in-module,import-error
     from kedro.cli.utils import KedroCliError  # pragma: no cover
 
 
 _VIZ_PROCESSES = {}  # type: Dict[int, multiprocessing.Process]
 
 _DEFAULT_KEY = "__default__"
 
-data = None  # pylint: disable=invalid-name
+_DATA = None  # type: Dict
+_CATALOG = None  # type: DataCatalog
```

Comment:
You can do `_DATA: Optional[Dict] = None` so you don't have to disable unused-import. Also `node` (lowercase) isn't a type, is it?

Reply:
https://github.com/quantumblacklabs/kedro-viz/pull/231/files#r466326373 fixed the unused-import thing :)